src/runtime/rpc/rpc_endpoint.h

```diff
@@ -135,8 +135,7 @@ class RPCEndpoint {
    * \param ctx_to The target context.
    * \param type_hint Hint of content data type.
    */
-  void CopyToRemote(void* from, size_t from_offset, void* to, size_t to_offset, size_t nbytes,
-                    TVMContext ctx_to, DLDataType type_hint);
+  void CopyToRemote(void* from_bytes, DLTensor* to, uint64_t nbytes);
```

Comment:
consistency: either use size_t  nbytes or uint64_t nbytes everywhere (RPCSession and RPCEndpoint)